cdap-ui/app/cdap/components/DataPrep/DataPrepSidePanel/DataPrepSidePanel.scss

```diff
@@ -144,15 +161,19 @@ $tab-header-height: 37px;
       .columns-tab-row {
         border-bottom: 1px solid #cccccc;
 
+        .row-header-checkbox {
+          cursor: pointer;
+          margin-right: 10px;
+        }
+
         .row-header {
           padding: 5px 10px;
-          cursor: pointer;
           white-space: nowrap;
           overflow: hidden;
           text-overflow: ellipsis;
 
-          &.expanded {
-            background-color: #dedede;
+          &.selected {
+            background-color: $element-selected-color;
```

Comment:
Just noticed this for some colors we have them separated out in a variable and some are not. Any specific reasons?